pandas/tests/window/test_rolling.py

```diff
@@ -98,6 +98,95 @@ def test_constructor_timedelta_window_and_minperiods(self, window, raw):
         tm.assert_frame_equal(result_roll_sum, expected)
         tm.assert_frame_equal(result_roll_generic, expected)
 
+    @pytest.mark.parametrize(
+        "dataframe,expected,window",
+        [
+            (
+                DataFrame({"A": [1, 2, 3], "B": [4, 5, 6]}),
```

Comment:
The `dataframe` and `series` parameters all look to be the same across the tests. You can bring them into the test directly (will be easier to see what's being

Reply:
Maybe not this test specifically, but the ones where applicable.